src/setup/askForCustomPort/askForCustomPort.ts

```
@@ -1,14 +1,21 @@
-  const { customPort } = await inquirer.prompt([
+  const { customPort } = await inquirer.prompt<{ customPort: string }>([
     {
       type: 'input',
       name: 'customPort',
       message:
         'Enter custom port for development server (leave blank for default 4321):',
-      default: 4321,
+      default: '4321',
+      validate: (input) => {
+        const port = Number(input);
+        if (isNaN(port) || port <= 0 || port > 65535) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)